### PR TITLE
refactor(perception_utils): add label util function

### PR DESCRIPTION
**Pull request by scepter914**
## Description

Add label util function to perception_util.
After this PR is merged, I'll replace some function like <https://github.com/autowarefoundation/autoware.universe/blob/main/perception/lidar_centerpoint/lib/ros_utils.cpp#L106> to util function.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: common/perception_utils/include/perception_utils/object_classification.hpp [hunk 1]
```diff
@@ -0,0 +1,41 @@
+
+// Copyright 2022 TIER IV, Inc.
```
Comment (kenji-miyake) [inline common/perception_utils/include/perception_utils/object_classification.hpp:2]: [style] An unnecessary empty line? :thinking: 

```suggestion
// Copyright 2022 TIER IV, Inc.
```

File: common/perception_utils/include/perception_utils/object_classification.hpp [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+#ifndef PERCEPTION_UTILS__OBJECT_CLASSIFICATION_HPP_
+#define PERCEPTION_UTILS__OBJECT_CLASSIFICATION_HPP_
+
+#include "autoware_auto_perception_msgs/msg/object_classification.hpp"
+
+namespace perception_utils
+{
+using autoware_auto_perception_msgs::msg::ObjectClassification;
+
+bool isVehicle(const uint8_t object_classification)
```
Comment (kenji-miyake) [inline common/perception_utils/include/perception_utils/object_classification.hpp:25]: I guess if you use `vehicle` then the bicycle is also treated as a vehicle.
I'd recommend asking native speakers for naming.

```suggestion
bool isCar(const uint8_t object_classification)
```

Comment (scepter914) [reply common/perception_utils/include/perception_utils/object_classification.hpp:24]: Thank you for thoughtful review.

I check the definition of vehicle like <https://www.nuscenes.org/nuscenes#data-annotation>.
So I fix isVehicle() function in [e6c22e2](https://github.com/autowarefoundation/autoware.universe/pull/1908/commits/e6c22e26d6986815c678839378f76bb3270b99c2).

In addition, I add new function isCarLikeVehicle() function in [2087bea](https://github.com/autowarefoundation/autoware.universe/pull/1908/commits/2087beae8a140127e9c2bf797468dc21b7d728c0),  consulting with @yukke42 .

**Review by yukke42 — APPROVED** [review]
LGTM

**Review by takayuki5168 — APPROVED** [review]
LGTM

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**